src/monitoring/index.ts

```diff
@@ -65,9 +67,74 @@ export default class MonitoringService {
     module: "monitoringService",
   })
 
+  private readonly REDIS_CONNECTION = {
+    host: config.get("bullmq.redisHostname"),
+    port: 6379,
+  }
+
+  private readonly queue = new Queue("MonitoringQueue", {
+    connection: {
+      ...this.REDIS_CONNECTION,
+    },
+    defaultJobOptions: {
+      removeOnComplete: true,
+      removeOnFail: true,
+      attempts: 3,
+      backoff: {
+        type: "exponential",
+        delay: 60000, // this operation is not critical, so we can wait a minute
+      },
+    },
+  })
+
+  private readonly worker: Worker<unknown, string, string> | undefined
```

Comment:
do you know how to? the way that i see this, this is a limitation of the library, as is allows jobs of any shape to be scheduled. in this case, we want to keep the type to unknown and actually we are not using the data at all (since now the invariant is that any job is a monitoring job, so start the scheduled driver)

Reply:
the library exposes typings on `Worker` and `Job`. 

```ts
  private readonly worker: Worker<number, number, string>
  this.worker = new Worker<number, number>
  job: Job<number, number>
```

these allowed me to type it correctly. since we're just using the queue as a cronjob, the value add from `unknown -> void` might not be that much. fine w/ either